net-p2p/deluge/deluge-2.1.1.ebuild

```diff
@@ -0,0 +1,132 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_SINGLE_IMPL=1
+inherit xdg distutils-r1 systemd
+
+DESCRIPTION="BitTorrent client with a client/server model"
+HOMEPAGE="https://deluge-torrent.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.deluge-torrent.org/${PN}"
+else
+	SRC_URI="http://download.deluge-torrent.org/source/$(ver_cut 1-2)/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="console gui libnotify sound webinterface"
+REQUIRED_USE="
+	libnotify? ( gui )
+	sound? ( gui )
+"
+
+# Tests segfault and require dev-python/pytest-twisted which is not in ::gentoo
+RESTRICT="test"
+
+BDEPEND="
+	${DISTUTILS_DEPS}
+	dev-util/intltool
+"
```

Comment:
@Kangie , oh no! We mixed up to variable during last change :(

```suggestion
REQUIRED_USE="
	${PYTHON_REQUIRED_USE}
	libnotify? ( gui )
	sound? ( gui )
"

# Tests segfault and require dev-python/pytest-twisted which is not in ::gentoo
RESTRICT="test"

BDEPEND="
	${DISTUTILS_DEPS}
	dev-util/intltool
"
```

We have to use `${PYTHON_REQUIRED_USE}` and have to delete `${DISTUTILS_DEPS}`.

Double sorry.